plugins/jira-dashboard/src/components/JiraTable/JiraTable.tsx

```diff
@@ -1,32 +1,75 @@
 import React from 'react';
 import Typography from '@mui/material/Typography';
-import { JiraDataResponse } from '@axis-backstage/plugin-jira-dashboard-common';
-import { ErrorPanel, Table } from '@backstage/core-components';
+import {
+  JiraDataResponse,
+  Project,
+} from '@axis-backstage/plugin-jira-dashboard-common';
+import {
+  ErrorPanel,
+  Table,
+  Link,
+  SupportButton,
+} from '@backstage/core-components';
 import { capitalize } from 'lodash';
 import { columns } from './columns';
+import { getJiraUrl } from '../../lib';
+import { Alert, AlertTitle } from '@material-ui/lab';
```

Comment:
This should be imported as instead, no need to add the "@material-ui/lab'" dependency.

`import Alert from '@mui/material/Alert';`